### DISCUSSION TITLE
DAI 2023-2024 - My Docker Compose application - Pirakas Anthon

### DISCUSSION THREAD

**Discussion by apirakas** [heig-vd-dai-course/heig-vd-dai-course#128] [Show and tell]
My repo : https://github.com/apirakas/DAI-Pratical_content_10

![GitHubContainerRegistry](https://github.com/heig-vd-dai-course/heig-vd-dai-course/assets/71434215/6b473477-5802-4b44-84e8-f7ea470777ef)

**Comment by ludelafo** [comment]
Everything looks good to me! I would still add a gitignore to avoid any data leaks if I add all files but forget to check with `git status`.
